webcomponents/tool/src/main/frontend/js/sui-picture-changer/sui-picture-changer.js

```diff
@@ -133,6 +133,8 @@ export class SuiPictureChanger extends SakaiElement {
         }
       }
       this.csrfToken = json.csrf_token;
+      // Update the profile image on the page
```

Comment:
I feel like this should happen at line 119

Reply:
Why?